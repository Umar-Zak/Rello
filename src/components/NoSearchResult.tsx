import React from "react";
import "styled-components"
import styled from "styled-components/native";
function NoSearchResult({text}: {text?: string}) {
    return (
       <Container>
        <Text>
           {text? text : "No results"}
        </Text>
       </Container>
    );
}

export default NoSearchResult;

const Container = styled.View`
    flex: 1;
    align-items: center;
    justify-content: center;
`

const Text = styled.Text`
    
`