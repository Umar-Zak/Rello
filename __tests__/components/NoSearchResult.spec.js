import React from "react";
import {render} from "react-native-testing-library"
import "jest-styled-components"
import NoSearchResult from "../../src/components/NoSearchResult"

describe("No Search Result", () => {

    it("Should render correctly", () => {
       const snapshot =  render(
            <NoSearchResult/>
        ).toJSON()

        expect(snapshot).toMatchSnapshot()
    })
})