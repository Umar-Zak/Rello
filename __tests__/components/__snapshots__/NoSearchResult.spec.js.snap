// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Search Result Should render correctly 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      [
        {},
      ]
    }
  >
    No results
  </Text>
</View>
`;
